doc/governance.rst

```diff
@@ -36,7 +36,7 @@ crucial to improve the communication in the project and limit the crowding
 of the issue tracker.
 
 Similarly to what has been decided in the `python project
-<https://devguide.python.org/triaging/#becoming-a-member-of-the-python-triage-team>`_,
+<https://devguide.python.org/triage/triaging/index.html>`_,
```

Comment:
this sounds good to me, but the `conf.py` change then need to be reverted :)

Reply:
Actually I think this may be the link we want (the anchor does still exist but in a different page somehow): https://devguide.python.org/triage/triage-team/#becoming-a-member-of-the-python-triage-team

Reply:
Working on that now. Sorry, a bit new to this so I'm trying to figure out how to properly revert that commit.  For some reason my local repo doesn't show the change I previously made.

Reply:
Working on that now.

Reply:
I removed the Devguide link from conf.py.  The pull request should be accurate and complete now.

Reply:
@jasonjg you might have missed this comment: https://github.com/scikit-learn/scikit-learn/pull/25035#discussion_r1032174102

Reply:
> https://devguide.python.org/triage/triage-team/#becoming-a-member-of-the-python-triage-team

Just checking before I make a commit.  Is the URL I quoted the one you want to use?

Reply:
Yes please :pray:.

I think this was the original intent of the link to point towards the Python Triage team policy (rather than a checklist of what to do for the Triage team)